src/easy/mod.rs

```diff
@@ -23,7 +23,7 @@
 use std::io;
```

Comment:
This file's documentation still talks about `FramedIo` despite it being removed.